README.md

```diff
@@ -1 +1,77 @@
 # Physlr: Construct a Physical Map from Linked Reads
+
+Physlr constructs a *de novo* physical map using linked reads from 10X Genomics or stLFR. This physical map can then be used to scaffold an existing assembly to yield chromosomal level contiguity.
+
+# Dependencies
+
+* [ntCard](https://github.com/bcgsc/ntCard)
+* [ntHits](https://github.com/bcgsc/ntHits)
+* GCC 5 or newer with OpenMP
+* Python 3.5 or newer and the following packages
+    * [community](https://python-louvain.readthedocs.io/en/latest/api.html)
+    * [networkx](https://networkx.github.io/)
+    * [numpy](https://numpy.org/)
+    * [scipy](https://www.scipy.org/)
+    * [sklearn](https://scikit-learn.org/stable/)
+    * [tqdm](https://tqdm.github.io/)
+
+Additionally, we recommend using pypy3 over regular python3 for speed.
+
+
+## Optional dependencies
+
+- [pigz](https://zlib.net/pigz/) for parallel gzip
+- [zsh](https://sourceforge.net/projects/zsh/) for reporting time and memory usage
+
+
+# Compiling Physlr from source
+
+```
+pip3 install --user git+https://github.com/bcgsc/physlr
+cd src
+make
+```
+
+# Running Physlr
+
+## Generating Physlr Physical Map with stLFR reads
+
+To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome.
+In this example, the linked reads and reference genome are called `linkedReads.fq.gz` and `reference.fa`, respectively. The linked reads are from stLFR so we specify `minimizer_overlap=stLFR` to use the default value for stLFR reads.
+
+```
+cd experiment
+ls
+linkedReads.fq.gz #stLFR   reference.fa
+bin/physlr-make physical-map lr=linkedReads ref=ref minimizer_overlap=stLFR
+```
+
+## Scaffolding a draft assembly with Physlr Physical Map
+
+To scaffold a draft assembly, you need linked reads from 10X Genomics or stLFR, and an existing assembly. In addition, to calculate Quast summary metrics for the Physlr scaffolded assembly, you need a reference genome.
+In this example, the linked reads, draft assembly, and reference genome are called `linkedReads.fq.gz`, `draft.fa`, `reference.fa`, respectively. The linked reads are from 10X Genomics so we specify `minimizer_overlap=10X` to use the default value for 10X Genomics reads.
+
+```
+cd experiment
+ls
+linkedReads.fq.gz #10X   reference.fa    draft.fa
+bin/physlr-make scaffolds lr=linkedReads ref=reference draft=draft minimizer_overlap=10X
+```
+
+See the help page for further information.
+```bin/physlr-make help```
```

Comment:
Why triple back ticks on one line? That's unusual. Usually it's triple back ticks for a block or single back ticks for inline.

Reply:
Didn't know there was a distinction. Will change to single back ticks